nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -154,24 +154,7 @@ let
 
   overrideConfig = let
     c = cfg.config;
-    requiresReadSecretFunction = c.dbpassFile != null || c.objectstore.s3.enable;
-    objectstoreConfig = let s3 = c.objectstore.s3; in optionalString s3.enable ''
-      'objectstore' => [
-        'class' => '\\OC\\Files\\ObjectStore\\S3',
```

Comment:
Hmm... Maybe we want to set this with `mkDefault` if the rest of objectstore is set? :thinking: